.scripts/get-interface-test-targets.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env -S uv run --script --no-project
+
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+#     "PyYAML",
+# ]
+# ///
+
+# Copyright 2025 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Collect the targets for running interface tests for the specified interface.
+
+Only outputs targets that there are test cases for by default.
+"""
+
+import argparse
+import json
+import logging
+import pathlib
+import re
+import subprocess
+import tempfile
+
+import yaml
+
+_REPO_ROOT = pathlib.Path(__file__).parent.parent
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(str(pathlib.Path(__file__).relative_to(_REPO_ROOT)))
```

Comment:
Is this just `.scripts`? Seems much clearer just to say `getLogger('.scripts')`.